src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py

```diff
@@ -251,6 +279,17 @@ def acr_build_task_update(cmd,  # pylint: disable=too-many-locals
 
     return build_task
 
+def acr_build_task_update_build(cmd,
+                                client,
+                                build_id,
+                                registry_name,
+                                is_archive_enabled=None,
+                                resource_group_name=None):
+    if is_archive_enabled is not None:
+        return client.update(resource_group_name=resource_group_name,
+                             registry_name=registry_name,
+                             build_id=build_id,
+                             is_archive_enabled=is_archive_enabled)
```

Comment:
I think you can just return directly without checking if `is_archive_enabled` is None or not, otherwise this command doesn't print anything if `is_archive_enabled` wasn't specified. Also add `is_archive_enabled` to `_params.py` with description if you haven't. Were you able to run this command and get expected results from server?

IMO, also rename this argument to `archive_enabled` or event `no_archive` (to be consistent with `no_push` or `no_logs`), but I would wait to see what others might say.